### ISSUE TITLE
Duplicated findings for `string-format` when there is more than one scope provided for single argument

### ISSUE THREAD

**Issue by zak-pawel** [mgechev/revive#1063]
**Describe the bug**
For `string-format` rule, when there is more than one scope provided for single argument there are duplicated findings in the results.
The number of findings is the same as the number of scopes provided in the configuration.

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```shell
# flags
revive -config revive.toml msg.go
```

```toml
# config file
[rule.string-format]
    arguments = [
        ["fmt.Errorf[0],errors.New[0]", "/^([^A-Z]|$)/", "must not start with a capital letter"],
    ]
```

`msg.go`:
```go
package main

import (
	"errors"
)

func ReturnError() error {
	return errors.New("This is an error.")
}
```

**Expected behavior**
A single finding is found.

**Expected behavior**
Two findings are there:
```
msg.go:8:20: must not start with a capital letter
msg.go:8:20: must not start with a capital letter
```

**Desktop (please complete the following information):**
 - OS: `Ubuntu 23.10`
 - Version of Go: `1.23.0`

**Comment by chavacava** [comment]
Hi @zak-pawel thanks for reporting the issue.

**Comment by zak-pawel** [comment]
@chavacava 
Great, thanks. It works as it should. 
When can we expect the next release? Several nice fixes have been merged recently.
